modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/idp/v1/IdPFailureTest.java

```diff
@@ -96,6 +102,19 @@ public void testGetIdPWithInvalidId() {
         validateErrorResponse(response, HttpStatus.SC_NOT_FOUND, "IDP-60002", "random-id");
     }
 
+    @Test
+    public void testInvalidSearchAllIdPs() throws XPathExpressionException {
```

Comment:
This is actually success test, let move to success test class.

Reply:
This is updated with be797d1079d847b27cc76440a6cb21ecb3476a5d